cmd/tarmak/cmd/cluster_ssh.go

```diff
@@ -2,18 +2,25 @@
 package cmd
 
 import (
+	"errors"
+
 	"github.com/spf13/cobra"
 
 	"github.com/jetstack/tarmak/pkg/tarmak"
 )
 
 var clusterSshCmd = &cobra.Command{
-	Use:   "ssh [instance alias] [optional ssh command]",
+	Use:   "ssh [instance alias] [optional ssh arguments]",
 	Short: "Log into an instance with SSH",
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return errors.New("expecting an instance aliases argument")
+		}
+		return nil
+	},
 	Run: func(cmd *cobra.Command, args []string) {
 		t := tarmak.New(globalFlags)
-		defer t.Cleanup()
-		t.SSHPassThrough(args)
+		t.Perform(t.SSHPassThrough(args[0], args[1:]))
```

Comment:
This is still  ssh CLI powered instead of golang native SSH package.

The following seems to break (Which is really useful for debugging):

```
tarmak-0.5.2 cluster ssh bastion -vvv
OpenSSH_7.6p1 Ubuntu-4ubuntu0.1, OpenSSL 1.0.2n  7 Dec 2017
debug1: Reading configuration data /home/christian/.tarmak-dev/csire-cluster/ssh_config
debug2: resolving "bastion" port 22
ssh: Could not resolve hostname bastion: No address associated with hostname
FATA[0000] exit status 255 
```

On master this looks like:

```
./tarmak_linux_amd64 cluster ssh bastion -vvv
ssh: Could not resolve hostname bastion: No address associated with hostname
FATA[0000] exit status 255
```